Components/Library.html

```diff
@@ -86,20 +87,26 @@ <h2 class="sidebar__heading-name">COMPONENTS</h2>
                     <a href="./modal/modal.html">
                         <li class="sidebar__link-item">MODAL</li>
                     </a>
-                    <a href="./slider/alert.html">
+                    <a href="../slider/slider.html">
                         <li class="sidebar__link-item">SLIDER</li>
                     </a>
+                    <a href="../rating/rating.html">
+                        <li class="sidebar__link-item">RATING</li>
```

Comment:
Be consistent with the class naming pattern. Either use - or _.

Reply:
Got it will be changing in the end!